workitem/link/type_repository.go

```diff
@@ -20,9 +20,10 @@ import (
 // WorkItemLinkTypeRepository encapsulates storage & retrieval of work item link types
 type WorkItemLinkTypeRepository interface {
 	Create(ctx context.Context, name string, description *string, sourceTypeID, targetTypeID uuid.UUID, forwardName, reverseName, topology string, linkCategory, spaceID uuid.UUID) (*app.WorkItemLinkTypeSingle, error)
-	Load(ctx context.Context, ID uuid.UUID) (*app.WorkItemLinkTypeSingle, error)
-	List(ctx context.Context) (*app.WorkItemLinkTypeList, error)
-	Delete(ctx context.Context, ID uuid.UUID) error
+	Load(ctx context.Context, spaceID uuid.UUID, ID uuid.UUID) (*app.WorkItemLinkTypeSingle, error)
+	LoadByID(ctx context.Context, ID uuid.UUID) (*app.WorkItemLinkTypeSingle, error)
```

Comment:
I created LoadByID to give support to links, comments and other elements that don't have any space context, just the work item ID.